### DISCUSSION TITLE
How to handle the custom md parsing capability

### DISCUSSION THREAD

**Discussion by RSS1102** [web-infra-dev/rspress#460] [Q&A]
我有一个自己的md渲染引擎可以解析某个md文本>(内置一个node api可以解析转换成html元素字符串，一个node api可以解析md文本的标题成成json文件)。我可以通过一些操作把这些解析的这些转换到rspress嘛？

**Comment by sanyuan0704** [comment]
1. 你的渲染引擎能力在当前的 rspress 的 markdown 编译器中能否实现？
2. 如果要转换 md 编译逻辑，可以在 rspress 配置中将 rspack 相关的 loader 移除，添加自定义的 loader 来接入自定义 md 渲染逻辑。

**Reply by RSS1102** [reply]
这个的一些解析和当前的markdown解析可能不一致，有些是自己的实现逻辑。

---
我尝试一下先，谢谢指点。
